### PR TITLE
remove the unnecessary  logs on validator

### PR DESCRIPTION
**Pull request by LegendWorrier**
• Removed unnecessary logs on validator(miner_client.py, miner_service.py, task_service.py)

### CODE REVIEW

File: neurons/validators/src/clients/miner_client.py [hunk 1]
```diff
@@ -137,10 +137,6 @@ async def await_connect(self):
                     )
                 if self.debounce_counter:
                     sleep_time = self.sleep_time()
-                    logger.info(
```
Comment (pyon12) [inline neurons/validators/src/clients/miner_client.py:140]: @LegendWorrier I think we need to keep this log

File: neurons/validators/src/services/task_service.py [hunk 2]
```diff
@@ -137,13 +125,6 @@ async def create_task(
                         )
                         return machine_spec, executor_info, 0
 
-                    logger.info(
```
Comment (pyon12) [inline neurons/validators/src/services/task_service.py:140]: @LegendWorrier I think we need to keep this log

Comment (pyon12) [inline neurons/validators/src/services/task_service.py:140]: @LegendWorrier @talentwebdev I think we need to keep this log

**Comment by pyon12** [comment]
@LegendWorrier I don't think we need to keep these 2 logs. please remove them.
image

**Comment by LegendWorrier** [comment]
• Fixed task_service.py of validator.

**Pull request merged**